ceph.spec.in

```diff
@@ -1433,11 +1433,9 @@ fi
 %{_bindir}/ceph-post-file
 %{_tmpfilesdir}/ceph-common.conf
 %if 0%{with python2}
-%dir %{python_sitelib}/ceph
```

Comment:
this change should be squashed into the previous one.

Reply:
fixed